packages/core/content-releases/shared/contracts/releases.ts

```diff
@@ -44,6 +44,7 @@ export declare namespace GetReleases {
     data: ReleaseDataResponse[];
     meta: {
       pagination?: Pagination;
+      countPendingReleases?: number;
```

Comment:
I think `pendingReleasesCount` would read better, because `countPendingReleases` sounds to me like an action (so a function)